routes/weather.ts

```diff
@@ -339,6 +324,10 @@ export const getWateringData = async function( req: express.Request, res: expres
  * @param useJson Indicates if the response body should use a JSON format instead of a format similar to URL query strings.
  */
 function sendWateringError( res: express.Response, error: CodedError, useJson: boolean = false ) {
+	if ( error.errCode === ErrorCode.UnexpectedError ) {
```

Comment:
Should we log all of the "Internal Errors" not just the node ones ?

Reply:
Let's discuss this point in Slack